### PR TITLE
Feature/write to package.json clone

### PR DESCRIPTION
**Pull request by simeg**
Mergable copy of PR #16.

### CODE REVIEW

**Comment by simeg** [comment]
Hi @ZeroX-DG.

I noticed you were having some issues so I solved it for you. See how this PR doesn't have any commits that are already merged in master? It doesn't conflict with the current code so we are able to merge it.

**There are two ways to resolve conflicts:**
1) Merge commits. This is the approach that GitHub promotes and it's the easiest one, it doesn't really require any understanding of git, but in my opinion it really clutters the commit history so I never use it.
2) Rebase and force push. This is the approach I use, but if you don't know what you are doing you can basically undo your commits = wasted work.

Here are the commands I ran to fix this:
```sh
cd gulpfile-generator/
git status # I am in master branch
git fetch origin # Fetch latest changes from master
git rebase origin/master # Rebase (apply) latest changes from origin (pushed code on GitHub) to my local master. I do this to make sure I have the latest changes.
git checkout -b feature/write-to-package.json-clone # Create a new branch
git remote add pr https://github.com/ZeroX-DG/gulpfile-generator.git # Add a reference to your repo. I do this to be able to fetch commits from your branch, and I name it "pr"
git fetch pr # Fetch all the branches and commits from your repo to my repo
git rebase pr/feature-write-to-package.json # I rebase the changes from your branch to my current branch, which means I'm putting all your commits into my branch
# Running the rebase command caused like 5 conflicts which I resolved, and git helped me with instructions
git log --oneline # I want to verify and make sure all the commits I'm interested in are in place. Everything looks good!
git push origin feature/write-to-package.json-clone # Push to MY new branch, which is basically a clone of your branch
```

The concept is simple really, but it's hard to explain. Maybe [this article](https://www.atlassian.com/git/tutorials/merging-vs-rebasing) is better at explaining.

File: gulpfile.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ gulp.task('test', ['set-test-env'], function(cb) {
 
     gulp.src('test/**/*.js')
         .pipe(plumber())
+        .pipe(istanbul())
```
Comment (simeg) [inline gulpfile.js]: What is this doing, and why is it needed?

File: package.json [hunk 2]
```diff
@@ -34,6 +34,7 @@
     "inquirer": "2.0.0"
   },
   "devDependencies": {
+    "eslint-plugin-html": "3",
```
Comment (simeg) [inline package.json:37]: Where is this used, and why is it needed?

Comment (ZeroX-DG) [reply package.json:37]: I have no idea. At first I encountered this error when I run test
```
> gulpfile-generator@1.3.3 test /home/zerox/Desktop/projects/gulpfile-generator
> NODE_ENV=test gulp

[08:17:15] Using gulpfile ~/Desktop/projects/gulpfile-generator/gulpfile.js
[08:17:16] Starting 'set-test-env'...
[08:17:16] Finished 'set-test-env' after 85 μs
[08:17:16] Starting 'json'...
[08:17:16] Finished 'json' after 8.48 ms
[08:17:16] Starting 'lint'...
[08:17:16] Finished 'lint' after 28 ms
[08:17:16] Starting 'test'...

events.js:160
      throw er; // Unhandled 'error' event
      ^
Error: Failed to load plugin html: Cannot find module 'eslint-plugin-html'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.resolve (internal/module.js:27:19)
    at Plugins.load (/home/zerox/Desktop/projects/gulpfile-generator/node_modules/eslint/lib/config/plugins.js:106:29)
    at Array.forEach (native)
    at Plugins.loadAll (/home/zerox/Desktop/projects/gulpfile-generator/node_modules/eslint/lib/config/plugins.js:146:21)
    at loadFromDisk (/home/zerox/Desktop/projects/gulpfile-generator/node_modules/eslint/lib/config/config-file.js:492:35)
    at Object.load (/home/zerox/Desktop/projects/gulpfile-generator/node_modules/eslint/lib/config/config-file.js:550:20)
    at Config.getLocalConfigHierarchy (/home/zerox/Desktop/projects/gulpfile-generator/node_modules/eslint/lib/config.js:228:44)
    at Config.getConfigHierarchy (/home/zerox/Desktop/projects/gulpfile-generator/node_modules/eslint/lib/config.js:180:43)
    at Config.getConfigVector (/home/zerox/Desktop/projects/gulpfile-generator/node_modules/eslint/lib/config.js:287:21)
npm ERR! Test failed.  See above for more details.

```
I resolved it by installing `eslint-plugin-html` using
```
yarn add eslint-plugin-html -D 
```

That's why it's there

File: gulpfile.js (hunk 1 above)
Comment (ZeroX-DG) [reply gulpfile.js]: Oh, I forgot to remove it, I was reading the gulp-istanbul documentation and I tested it and I forgot to remove it, sorry :)

Comment (simeg) [reply gulpfile.js]: No worries, I'll take care of it.

File: package.json (hunk 2 above)
Comment (simeg) [reply package.json:37]: Strange, I can run tests without it being present. But if you can't let's leave it there.

**Pull request merged**